src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1611,6 +1575,42 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserInSectionBy
                         visibilityTable, responseStatus, roster, responseComments);
     }
 
+    private Map<String, String> initializeParamsWithSelectedSectionDetail(String sectionDetail) {
```

Comment:
How about using `giver`, `reciever`, `giverOrReciever`, `giverAndReciever` ? Future developers may be confused with `either` and `both`